### DISCUSSION TITLE
How to display a thumbnail in a collection (not Media)

### DISCUSSION THREAD

**Discussion by bhofstaetter** [payloadcms/payload#9684] [Q&A] [answered]
Hi,

I have a collection with an image upload field. How can I display the thumbnail of this field, in a cell on the collection overview page (just like in the media collection)? Do I need to write a custom cell component for this or is there a build in way?

Thank's a lot!

**Comment by mouktardev** [comment, marked as answer]
In your Media Collection upload options just add `displayPreview:true,` and any collection  that have relations to media will have a thumbnail similar.

```ts
export const Media: CollectionConfig = {
  upload: {
      displayPreview:true,  
   }
}
```
found it here in the [docs](https://payloadcms.com/docs/upload/overview#collection-upload-options) 
also reference here #5015

**Reply by bhofstaetter** [reply]
cool thank you very much, unfortunately I didn't understand what this option was supposed to do when I read it, I thought it was for the preview within the upload field.

however, if my upload field is not at the top level, but nested in a group field, for instance. Is there an “easy” way to still use it as a preview?

If I want to get rid of the file name next to the image preview, I have to overwrite the cell component, right?

**Reply by pevlabs** [reply]
I too have the same requirement to suppress the image name after enabling the image thumbnail. I tried adding displayName:false after adding displayPreview:true as a shot in the dark. Appears to have been too wishful.
